certauth_test.go

```diff
@@ -59,8 +97,12 @@ var testHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request)
 })
 
 func TestMiddleWare(t *testing.T) {
-	a := certauth.NewAuth(certauth.Options{
-		AllowedOUs: []string{"someunit", "bestunit"},
+	allowedCert := fakeCertChain([]string{"endpoint"}, "foo.com")
+	failedCert := fakeCertChain([]string{"site"}, "foo.com")
+
+	auth := certauth.NewAuth(certauth.Options{
+		AllowedOUs: []string{"endpoint", "titan"},
+		AllowedCNs: []string{"foo.com"},
 	})
 
 	mux := http.NewServeMux()
```

Comment:
dun think we need the mux here .. I was going to go this way, but decided not too.. just some dead code now I think.